controllers/chat/group/groupAddMembers.js

```diff
@@ -15,6 +17,8 @@ const groupAddMembers = async (req, res) => {
   const {userId} = req;
   const {channel_id, members} = req.body;
 
+  const ownUser = await UsersFunction.findUserById(User, userId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `UsersFunction.findUserById` function is used here to find the user by their ID. However, there's no error handling in case the user is not found or an error occurs during the process. This could lead to unexpected behavior later on if `ownUser` is undefined.
```diff
- const ownUser = await UsersFunction.findUserById(User, userId);
+ const ownUser = await UsersFunction.findUserById(User, userId);
+ if (!ownUser) {
+   throw new ErrorResponse('User not found', 404);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->